openhtf/io/frontend/src/app/utils.ts

```diff
@@ -41,6 +41,7 @@ export class SimplifyStatus extends MapPipe {
     'WAITING_FOR_TEST_START': 'WAITING',
     'STOP_WAIT': 'WAITING',
     'TIMEOUT': 'ERROR',
+    'ABORTED': 'ERROR',
```

Comment:
Ah. The confusion is because this pipe is a little overloaded. It simplifies status, sometimes for more concise human readability and sometimes for color lookups. To make things extra confusing (but more simple for operators), test outcomes and test status are used to fill in the same spot in the UI (and determine the color of the test headers). Outcomes are preferred because if there is an outcome, that's the most import piece of summary info to display. If the test isn't finished yet, we display a simplified version of the status instead. Which is why I say this could all use a cleanup, but it should actually work the way it stands.